src/components/GetStarted/Faq.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ReactNode, useState, useCallback } from 'react'
+// @ts-ignore
+import { ButtonIcon, GU, RADIUS, useLayout, useTheme } from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { fontWeight } from '../../style/font'
+import arrow from '../../assets/arrow.svg'
+
+const AnimatedDiv = animated.div
+
+type FaqProps = {
+  content: ReactNode
+  expansion: ReactNode
+}
+
+type ToggleButtonProps = {
+  onClick: () => void
+  opened: boolean
+}
+
+function Faq({ content, expansion }: FaqProps): JSX.Element {
+  const [opened, setOpened] = useState(false)
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const compactMode = layoutName === 'small'
+
+  const toggleButton = useCallback(() => {
+    setOpened((opened) => !opened)
+  }, [])
+
+  return (
+    <div
+      css={`
+        position: relative;
+        margin-bottom: ${RADIUS}px;
+      `}
+    >
+      <div
```

Comment:
I think it would be better UX if this entire element was clickable, we can use `ButtonBase` here instead of the div and attach the `toggleButton` handler.